frontend/src/reducers/ActionReducers.ts

```diff
@@ -30,7 +30,7 @@ export default function setAction(state : State, action: Action) {
                 }
             }
         case ActionTypes.addItem:
-            let newItems = state.store.items;
+            let newItems = [...state.store.items];
```

Comment:
I ran into an issue where this action wasn't updating because ```newItems``` had the same reference as ```state.store.items``` so React's shallow compare didn't catch the update.